sdk/src/main/java/co/omise/android/models/Items.kt

```diff
@@ -0,0 +1,16 @@
+package co.omise.android.models
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class Item(
+    val sku: String? = null,
+    val category: String? = null,
+    val name: String? = null,
+    val quantity: String? = null,
+    val amount: String? = null,
+    val item_uri: String? = null,
+    val image_uri: String? = null,
```

Comment:
Should we use camelCase with jsonProperty like Billing and Shipping?

Reply:
Changed -> https://github.com/omise/omise-android/pull/239/commits/a0220bf3de209bf0ae40408ddcefe8b2bad8e372